index.markdown

```diff
@@ -472,6 +472,18 @@ are likely to just disappoint the winning team. When organizations/individuals
 wish to pursue further development of a hack they should speak to the winning
 team separately.
 
+### Attitude to prize winners
+
+Those who have won a prize at a hack day should not be expected to provide any
+ongoing service or favours to sponsors or organisers. Some hack day sponsors
+have been known to ask prize winners to write promotional blog posts for them.
+This kind of behaviour from sponsors can make quite prize winning participants
+quite uncomfortable and unlikely to wish to participate in future events.
+
+What may seem natural to outgoing marketers and CEOs may feel a lot less
+pleasant for more shy and introverted (especially new) developers, designers
+etc.
```

Comment:
I like the sentiment here but I feel like it's redundant as I think a prize should be for winning a hack, and nothing more, A prize should be a prize, and come without conditions.